### ISSUE TITLE
Storefront child theme issue #2: Accessibility/ legibility issues

### ISSUE THREAD

**Issue by ghost** [woocommerce/woocommerce-gutenberg-products-block#3830]
**I made a comprehensive analysis and tested all the themes listed in the [Storefront Themes ](https://woocommerce.com/product-category/themes/storefront-child-theme-themes/?categories=Themes&collections=theme&page=1&orderby=date) in WooCommerce website.**

You can check the [Issues map I've created in Figma](https://www.figma.com/file/vm7kZzr5uIFwacpKKZzMb5/Storefront-child-themes?node-id=1%3A142), where you will find all the issues divided by theme.

You will find in GitHub different issues where I grouped similar/ same problems and created different issues for each problem group.

__________________________________________________________________________________________________________

# Issue
Some accessibility/ legibility issues can be found across several themes. 

# Bookshop
(Child theme)
## 1. The delete button icon when adding a coupon has poor legibility.

![Frame 208](https://user-images.githubusercontent.com/69914963/107651672-8a6c2180-6c5e-11eb-839d-dd9f31018ae3.png)

### Contrast test
<img width="875" alt="Screen Shot 2021-02-11 at 11 45 40" src="https://user-images.githubusercontent.com/69914963/107651796-ae2f6780-6c5e-11eb-8dac-63a8b477d29b.png">

### Solution
We should remove unnecessary design elements that detriments legibility and make the delete icon a neutral color. We should apply this solution to every theme.
![Frame 210](https://user-images.githubusercontent.com/69914963/107652533-7248d200-6c5f-11eb-8e0a-99b9751d31ab.png)

## 2. There is a shadow under the text and the icons. This affects legibility.
![Frame 207](https://user-images.githubusercontent.com/69914963/107652767-afad5f80-6c5f-11eb-8173-0202716f6f5a.png)

### Solution
We should remove text shadows and icon shadows.

# Boutique
(Child theme)
## 1. Credit cards background should be lighter to avoid legibility issues with different card colors.

![Frame 193](https://user-images.githubusercontent.com/69914963/107653319-40843b00-6c60-11eb-9850-fc7d1d5248be.png)

### Solution
We should change the background color to a lighter/ neutral one to avoid having this issue and apply the style across all themes.
Sugested color: #EAEAEA

## 2. Order Summary badge numbers are not visible
![Frame 195](https://user-images.githubusercontent.com/69914963/107653774-c43e2780-6c60-11eb-88c0-ea5cc0d36131.png)

### Solution
We should use a lighter grey or just white for the number and the outline. 

# Deli
(Child theme)
## 1. Credit cards background should be lighter to avoid legibility issues with different card colors.

![Frame 198](https://user-images.githubusercontent.com/69914963/107654103-167f4880-6c61-11eb-884c-ee01350c48dd.png)

### Solution
We should change the background color to a lighter/ neutral one to avoid having this issue and apply the style across all themes.
Sugested color: #EAEAEA

## 2. Order Summary badge numbers are not visible
![Frame 211](https://user-images.githubusercontent.com/69914963/107654139-20a14700-6c61-11eb-85d4-51e7727a3793.png)

### Solution
We should use a lighter color or just white for the number and the outline. 

# Hotel
(Child theme)
## 1. "Proceed to checkout" button doesn't have a background.

![Frame 109](https://user-images.githubusercontent.com/69914963/107655306-3400e200-6c62-11eb-921a-98b18029c704.png)

# Outlet
(Child theme)
## 1. Buttons are not visible. It seems that they are not loading their styles. (Apply coupon, Proceed to checkout, Place order)

![Frame 112](https://user-images.githubusercontent.com/69914963/107656167-a70a5880-6c62-11eb-8d96-15cb840fb759.png)
![Frame 115](https://user-images.githubusercontent.com/69914963/107656169-a7a2ef00-6c62-11eb-8dd2-600a7c22cce8.png)
![Frame 116](https://user-images.githubusercontent.com/69914963/107656175-a83b8580-6c62-11eb-9ac4-f6e03db47059.png)

# Matthew
## 1. Check icon is not legible
![Frame 212](https://user-images.githubusercontent.com/69914963/107656397-e042c880-6c62-11eb-9daf-0319e59a826d.png)

### Solution
We should change the check icon to white.

**Comment by nerrad** [comment]
@Aljullu based on your work for styles in the blocks, do the issues mentioned here look like changes we should try to enforce in blocks css, or something addressed at the theme level (I'm leaning towards the latter)?

**Comment by Aljullu** [comment]
I think all issues listed here need to be fixed at the theme level. In most cases they are conflicts between the customizer default colors and other theme styles, I don't think we can do anything from WC Blocks.

Only exception is Bookshop's "There is a shadow under the text and the icons.", In this case I think it makes sense to enforce buttons that we want to be shown as text not to have text shadow. ~I will open a PR in Blocks.~ PR opened in Blocks: #3846.

**Comment by Aljullu** [comment]
> Matthew

Looking at the source code, it looks like it was intentional to change the checkbox colors. So that might not be a bug.

**Comment by nerrad** [comment]
I moved this report into individual issues:

### Bookshelf

- Fix delete icon style for coupon tags in Cart and Checkout Blocks (34-gh-woocommerce/bookshop)

### Boutique

- Lighten background color for payment method icons in the Cart block (https://github.com/woocommerce/boutique/issues/81)
- Fix order summary quantity badge numbers styling for cart and checkout blocks used with the theme. (https://github.com/woocommerce/boutique/issues/82)


### Deli

- Lighten background color for payment method icons in the Cart block (https://github.com/woocommerce/deli/issues/57)
- Fix order summary quantity badge numbers styling for cart and checkout blocks used with the theme. (https://github.com/woocommerce/deli/issues/58)

### Hotel

- Fix missing background for "Proceed to Checkout" button in Cart block (45-gh-woocommerce/hotel)

### Outlet

-  Fix button styling in Cart and Checkout blocks (45-gh-woocommerce/outlet)
